### PR TITLE
feat(app-start): Add Android app start-related spans

### PR DESCRIPTION
**Pull request by narsaynorath**
These spans are nested operations under the app.start.* spans on Android. Start collecting them for the Sentry product so we have some data before moving into the default config.

Note: This only adds the application portion of the spans. We are currently awaiting a new span that tracks the system portion of the app start and I will add that as another span op afterwards.

These spans are listed out [here](https://www.notion.so/sentry/70c42197962a41198992ae89383ceb57?v=af03325d6d3e4168bdb13bd3269fe3f1&pvs=4) for further information:

### CODE REVIEW

File: relay-event-normalization/src/normalize/span/description/mod.rs [hunk 1]
```diff
@@ -82,6 +82,26 @@ pub(crate) fn scrub_span_description(span: &Span) -> Option<String> {
                 // They are low-cardinality.
                 Some(description.to_owned())
             }
+            ("contentprovider", "load") => {
```
Comment (narsaynorath) [inline relay-event-normalization/src/normalize/span/description/mod.rs:85]: @jjbayer You pointed me to this file initially, is this the only change I need to make to have these new span ops ingested internally?

Comment (jjbayer) [reply relay-event-normalization/src/normalize/span/description/mod.rs:85]: You'll also want to extend the `MOBILE_OPS` constant in `defaults.rs`: https://github.com/getsentry/relay/blob/7606bce0845179da2799179aa6cea855d3859b4d/relay-dynamic-config/src/defaults.rs#L20-L21

Comment (narsaynorath) [reply relay-event-normalization/src/normalize/span/description/mod.rs:85]: Sounds good, I was thinking about doing that but I was worried adding the entry to `default.rs` would enable it incorrectly for all orgs. I've added them now and also regenerated some snapshots with these new ops

Comment (jjbayer) [reply relay-event-normalization/src/normalize/span/description/mod.rs:85]: That is a good point! I just merged https://github.com/getsentry/relay/pull/2924, so the `Feature::SpanMetricsExtractionAllModules` flag and only extend the mobile ops conditionally.

**Comment by narsaynorath** [comment]
@jjbayer Just a heads up I've addressed your comment in this PR and implemented the change we discussed to update the mobile ops conditionally. Totally open to seeing alternative implementations if you have suggestions on how to do this part better, but I found a way to do it in a single block 👍

**Review by jjbayer — APPROVED** [review]

**Comment by jjbayer** [comment]
> @jjbayer Just a heads up I've addressed your comment in this PR and implemented the change we discussed to update the mobile ops conditionally. Totally open to seeing alternative implementations if you have suggestions on how to do this part better, but I found a way to do it in a single block 👍

@narsaynorath looks good to me. There's definitely a way to do it with iterator chaining, but IMO not worth the effort and it would not increase readability.

**Pull request merged**